src/js/api.js

```diff
@@ -8,23 +8,55 @@ function makeResponder(cb){
   }
 }
 
+function getHandler(url, cb, suffix){
+  if(typeof url === 'function'){
+    cb = url;
+    url = makeUrl(parseLocation(), suffix);
+  }
+
+  superagent.get(url).end(makeResponder(cb));
+}
+
+function postHandler(url, cb, suffix){
+  if(typeof url === 'function'){
+    cb = url;
+    url = makeUrl(parseLocation, suffix);
+  }
+
+  superagent.post(url).end(makeResponder(cb));
+}
+
+function makeUrl(obj, suffix){
```

Comment:
Yeah, sorry. 😦 
I meant showing a difference between private and public. So the difference between `makeUrl` and `makeResponder`, `getHandler`, etc.